modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -200,7 +205,12 @@ class NDB_Form_user_accounts extends NDB_Form
 
         ///get the value for additional_user_info flag
         $config =& NDB_Config::singleton();
-        $additional_user_info = $config->getSetting('additional_user_info');
+        try {
```

Comment:
This shouldn't have a try/catch block.. if the setting is missing, it needs to be added to the schema.